finance.md

```diff
@@ -0,0 +1,69 @@
+# Financial Accounts and Practices
+
+_Note: Note all of the following have been implemented yet, the Finance WG is working on it_
+
+This document outlines finance and accounting practices of Hypha Worker Co-operative, including:
+
+- Listing of accounts at our financial institutions
+- Process for invoicing clients and accepted methods of bill payments
+- Process for payroll and expense reimbursement to Members
+- Tools and procedures for bookkeeping, accounting, and tax remittances
+
+## Organizational structure
+
+Hypha Worker Co-operative is a worker co-operative incorporated in Ontario, Canada. The business has seven Founding Members, four of which are First Directors with signing authority. As a non-share-capital worker co-operative, we are a non-profit organization, and every Member holds both a Worker and an Owner relationship with the business.
+
+## Financial accounts
+
+Our main financial institution is **Desjardins** and our main funds are stored in a business chequing account, where all outgoing transactions require 2 signatures from Members of the Finance WG.
+
+Hypha is expected to conduct business with clients from other countries and is ready to [accept funds in CAD, USD, and EUR](https://transferwise.com/help-smart/11/getting-started/2571907/what-currencies-do-you-support). We use **TransferWise** to minimize transaction and currency exchange fees. Our Transferwise account has local accounts in Canada, United States, and Europe, so we can bill clients as a local financial institution in their local currency. Our TransferWise account does not require multiple signatures, and the Finance WG is responsible for transfering money into our main Desjardins account when accumulated funds exceed a CAD 10,000 threshold.
+
+## Invoicing a client
+
+Hypha charges clients on a sliding scale, and projects may be billed hourly, weekly, monthly, or per project, depending on the specifics of each project. For all projects, regardless of how a customer is charged, Members log their hours using **Clockify**. For clients and billing models that require timesheets, Clockify is used to generate these timesheets to calculate the amount to invoice each period.
+
+### Generating timesheets
```

Comment:
Once we approve general flow, can we count screenshots as part of done-done, before we merge this?

Reply:
See my larger comment that maybe we merge this earlier and do refinement over time. I think a monolithic long standing PR feels harder to merge over time